README.rst

```diff
@@ -99,13 +99,13 @@ In order to develop or running the tests, you can do:
 
 .. code:: bash
 
-   . venv/bin/activate
+   source venv/bin/activate
 
 4. Install dependencies
 
 .. code:: bash
 
-    pip -r requirements.txt -r requirements-dev.txt
+    pip install -r requirements.txt -r requirements-dev.txt
```

Comment:
```
(venv)➜ python-readchar git:(master) ✗ pip -rrequirements.txt

Usage:   
  pip <command> [options]

no such option: -r
(venv)➜ python-readchar git:(master) ✗ pip --version
pip 1.5.6 from /[..]/python-readchar/venv/lib/python2.7/site-packages (python 2.7)
```

I don't know what version of pip you're using, but it should absolutely be necessary as far as I can tell